enterprise/core-edge/src/main/java/org/neo4j/coreedge/raft/replication/tx/ReplicatedTransactionStateMachine.java

```diff
@@ -45,6 +45,7 @@
 
 public class ReplicatedTransactionStateMachine<MEMBER> implements StateMachine
 {
+    static final byte HEADER_ID = 0;
```

Comment:
Maybe these constants should be kept in an enum near the TransactionHeaders class, since the namespace is quite small, and it would be unfortunate to accidentally introduce clashes.